src/custom/LearningCard/LearningCard.tsx

```diff
@@ -35,43 +31,17 @@ interface Props {
   path?: string;
   courseCount: number;
   courseType: string;
-  orgId?: string;
-  modalContent?: string;
 }
 
-const OptionalLink: React.FC<React.PropsWithChildren<{ path?: string; isExternal?: boolean }>> = ({
-  path,
-  children,
-  isExternal
-}) => {
+const OptionalLink: React.FC<React.PropsWithChildren<{ path?: string }>> = ({ path, children }) => {
   if (!path) {
     return <>{children}</>;
   }
 
-  return (
-    <CardLink
-      href={path}
-      target={isExternal ? '_blank' : undefined}
-      rel={isExternal ? 'noopener noreferrer' : undefined}
-    >
-      {children}
-    </CardLink>
-  );
+  return <CardLink href={path}>{children}</CardLink>;
 };
 
-const LearningCard: React.FC<Props> = ({
```

Comment:
lets rename this component to ContentOverviewCard , and the tutorial prop to curriculaContent and use the types from schemas .  couse count to unitCount  and word "courses" to units . the create new content needs to be separate content only reusing the base card styling and not a part of this component .

Reply:
i guess these changes need to be done in cloud ui though